openfisca_core/scripts/run_test.py

```diff
@@ -34,7 +35,7 @@ def main():
         try:
             country_package = importlib.import_module(args.country_package)
         except:
-            print('ERROR: `{}` does not seem to be a valid Openfisca country package.'.format(args.country_package))
+            print('ERROR: {} does not seem to be a valid Openfisca country package.'.format(args.country_package))
```

Comment:
It's usually better to keep quotes around an interpolated string, as it shows a potential empty string.